boards/arm/stm32l4/nucleo-l432kc/src/nucleo-l432kc.h

```diff
@@ -129,6 +129,16 @@ extern struct spi_dev_s *g_spi2;
 int stm32l4_gpio_initialize(void);
 #endif
 
+/****************************************************************************
+ * Name: stm32l4_spiregister
+ *
+ * Description:
+ *   Called to register spi character driver of initialized spi device for the Nucleo-L432KC board.
```

Comment:
Hi @Dictor please break this line to become <= 78 chars per line. Also is a good idea to run: ./tools/checkpatch.sh -f path/to/file/you/are/modifying.c :-)